app/utils/repo_fetcher.py

```
@@ -0,0 +1,97 @@
+from typing import Any, Callable, List, Protocol, Tuple
```

Ruff (0.11.9)

`typing.Any` imported but unused

Remove unused import: `typing.Any`

(F401)